Tasks/NodeTool/nodetool.ts

```diff
@@ -131,8 +135,28 @@ async function acquireNode(version: string): Promise<string> {
 
     let downloadUrl = 'https://nodejs.org/dist/v' + version + '/' + urlFileName;
 
-    let downloadPath: string = await toolLib.downloadTool(downloadUrl);
+    let downloadPath: string;
+
+    try 
+    {
+        downloadPath = await toolLib.downloadTool(downloadUrl);
+    } 
+    catch (err)
+    {
+        for (let errorKey of Object.keys(err))
```

Comment:
nit, can avoid loop by checking truthy on the property, e.g.: ` if (err['httpStatusCode'] && err['httpStatusCode'] == '404')`